yarn.lock

```diff
@@ -3130,14 +3130,22 @@
     "@types/history" "*"
     "@types/react" "*"
 
-"@types/react@*", "@types/react@^16.6.3", "@types/react@^16.8.8":
+"@types/react@*", "@types/react@^16.8.8":
   version "16.8.24"
   resolved "https://registry.yarnpkg.com/@types/react/-/react-16.8.24.tgz#8d1ea1fcbfa214220da3d3c04e506f1077b0deac"
   integrity sha512-VpFHUoD37YNY2+lr/+c7qL/tZsIU/bKuskUF3tmGUArbxIcQdb5j3zvo4cuuzu2A6UaVmVn7sJ4PgWYNFEBGzg==
   dependencies:
     "@types/prop-types" "*"
     csstype "^2.2.0"
 
+"@types/react@^16.9.11":
+  version "16.9.11"
```

Comment:
Having two versions of the same module can cause problems and this doesn't fully upgrade to the new version. All of the above dependencies for `@types/react` can match 16.9.11, so you should consolidate them all to use this one.

Reply:
Now this PR doesn't affect anything, because it's already loading 16.8.3. This also doesn't address the issue, which was for upgrading to at least 16.8.25.